apps/core/components/sectorDetails/ItinerarySectorDetails.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import * as React from 'react';
+import { Card, StyleSheet } from '@kiwicom/universal-components';
+import { createFragmentContainer, graphql } from '@kiwicom/margarita-relay';
+import {
+  ItineraryTypeRenderer,
+  SectorDetailsWrapper,
+} from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import ItineraryOneWay from '../ItineraryCard/itineraryDetail/ItineraryOneWay';
+import ItineraryReturn from '../ItineraryCard/itineraryDetail/ItineraryReturn';
+// @TODO - gemerated type seems to be broken - import type { ItinerarySectorDetails_data as ItineraryType } from './__generated__/ItinerarySectorDetails_data.graphql';
+
+type Props = {|
+  +data: any, // @TODO - cannot use genereated ?ItineraryType which seems to be broken,
+|};
+
+function ItinerarySectorDetails(props: Props) {
+  return (
+    <Card style={styles.container}>
+      <SectorDetailsWrapper>
+        <ItineraryTypeRenderer
+          typename={props.data?.__typename}
+          oneWayComponent={<ItineraryOneWay itinerary={props.data} />}
+          returnComponent={<ItineraryReturn itinerary={props.data} />}
+        />
+      </SectorDetailsWrapper>
+    </Card>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    marginBottom: parseFloat(defaultTokens.spaceSmall),
+  },
+});
+
+export default createFragmentContainer(ItinerarySectorDetails, {
+  data: graphql`
```

Comment:
the reason relay deprecated using graphql directly was to force us programmers to use better naming than calling everything `data`. Lets give it a name like itineraries instead 😊

Reply:
True 😊. Updated even for parent container.